StockPriceScraper/Program.cs

```diff
@@ -117,7 +117,7 @@ static async Task DownloadWebsitesAsync()
             // create list of urls to download
             foreach (string stockName in stockNames)
             {
-                var stockUrl = String.Format(stockBaseUrl, stockName);
+                var stockUrl = string.Format(stockBaseUrl, stockName);
```

Comment:
Why lower case? String is weird to me, because String does normally not exist as base class. In c# it looks like it does.

Reply:
[https://stackoverflow.com/questions/7074/what-is-the-difference-between-string-and-string-in-c](https://stackoverflow.com/questions/7074/what-is-the-difference-between-string-and-string-in-c)